addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -212,12 +304,13 @@ public void addDeviceList(DeviceModel model) {
     }
 
     /**
-     * Builds a {@link ThingUID} from a device model. The UID is build from
-     * the {@link BindingConstants#BINDING_ID} and value of
+     * Builds a {@link ThingUID} from a device model. The UID is build from the
+     * {@link BindingConstants#BINDING_ID} and value of
      * {@link DeviceModel#getProductName()} in which all characters NOT matching
      * the regex [^a-zA-Z0-9_] are replaced by "_".
      *
-     * @param device Discovered device model
+     * @param device
```

Comment:
It does not do that on my machine with the ESH profile, this should be a proper export of the profile:
[formatter.xml.txt](https://github.com/openhab/openhab2-addons/files/977299/formatter.xml.txt)

\* do rename it to .xml